apps/api/src/services/mongodb_handler.py

```diff
@@ -0,0 +1,119 @@
+import asyncio
+import os
+from enum import Enum
+from logging import getLogger
+from typing import Any, Mapping, Optional, Union
+
+from bson import CodecOptions
+from motor.core import AgnosticCollection, AgnosticDatabase
+from motor.motor_asyncio import AsyncIOMotorClient
+from pydantic import BaseModel, Field
+from pymongo.results import InsertOneResult, UpdateResult
+
+log = getLogger(__name__)
+
+STAGING_ENV = os.getenv("DEPLOYMENT") == "STAGING"
+
+MONGODB_URI = os.getenv("MONGODB_URI")
+MONGODB_CLIENT = AsyncIOMotorClient(MONGODB_URI)
+MONGODB_CLIENT.get_io_loop = asyncio.get_event_loop
```

Comment:
Thought: do we know if this workaround is still needed when running on Vercel?

Reply:
I removed it and tested the deployment. It still works, so we no longer need it.